pydm/widgets/enum_combo_box.py

```diff
@@ -46,9 +49,22 @@ def set_items(self, enums):
         new_enum_strings : tuple
```

Comment:
Doesn't match the function call.

Reply:
If you don't have a strong objection about it, I'd like to keep the param name as new_enum_strings as the method expects a tuple of strings. "enums" is ambiguous.

Reply:
@hhslepicka? This will break anything calling this as a `kwarg` from prior versions. This is not advisable but still supported
```python
combo.set_items(enums=('a', 'b', 'c'))
```
I do understand that the new name is more intuitive however.

Reply:
@hmbui, @teddyrendahl is right and with 1.0 we mentioned that we would not break the widgets API and since this does I would prefer not to change that at this moment. Please add a comment somewhere with a TODO for a possible 2.0 version in which we change that but for now please revert the parameter name. And also, please add the docstring to clarify the ambiguity.

Reply:
No problem. I didn't know 'enums' was a positional param in the old version. I'll change the name back to 'enums'.

Reply:
The problem is that we can have people in the wild using the parameter by name and it will break if we change it. Thank you for looking into that. :+1: